openshift-knative-operator/pkg/serving/extension.go

```diff
@@ -7,30 +7,38 @@ import (
 
```

Comment:
specific unit tests for this would be great.

Reply:
For?

Reply:
The entire behavior ideally.

Reply:
ok.